asset/cloud-client/src/test/java/com/example/asset/QuickStartIT.java

```diff
@@ -45,33 +43,29 @@
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
 public class QuickStartIT {
-  private static final String bucketName = UUID.randomUUID().toString();
+  private static final String bucketName = "java-docs-samples-testing";
```

Comment:
Can we use an environment var for this?

Reply:
Can anyone who is not a googler still run this integration test?